src/algebra/big_operators/finprod.lean

```diff
@@ -266,6 +270,8 @@ by rw [← mem_mul_support, finprod_eq_mul_indicator_apply, mul_indicator_mul_su
   ∏ᶠ a ∈ s, f a = ∏ᶠ a, mul_indicator s f a :=
 finprod_congr $ finprod_eq_mul_indicator_apply s f
 
+lemma finprod_eq_finprod_mem (f : α → M) : ∏ᶠ i, f i = (∏ᶠ i ∈ (mul_support f), f i) := by simp
```

Comment:
This lemma seems redundant in the face of `finprod_mem_mul_support`; especially since `finprod_mem_mul_support` as a name is precisely this lemma statement.

Reply:
I removed it.